api/src/main/java/module-info.java

```diff
@@ -0,0 +1,3 @@
+open module nuvotifier.nuvotifier.api {
```

Comment:
```suggestion
open module com.vexsoftware.votifier {
```
This ought to follow the conventions for module names.

A module name is not like artifact coordinates - two jars containing the same packages should have the same module name. NuVotifier would have the same module name as Votifier theoretically would.